sympy/physics/wigner.py

```diff
@@ -1084,8 +1084,8 @@ def wigner_d_small(J, beta):
         for j, Mj in enumerate(M):
 
             # We get the maximum and minimum value of sigma.
-            sigmamax = max([-Mi-Mj, J-Mj])
-            sigmamin = min([0, J-Mi])
+            sigmamax = min([J-Mi, J-Mj])
```

Comment:
What is the motivation behind this change? Can you point me to some failing examples ... or maybe resources showing this as the expected value?

Reply:
The formula for the Wigner small d-matrix can be found in section 4.3.1, equation (2) of [https://doi.org/10.1142/9789814415491_0005](url) or by examining the implemented code. In this formula, the sum should only include values where all factorials are non-negative, meaning that the binomial coefficients are not equal to zero. However, in the current code, this condition is not satisfied, leading to terms of the form 0/0, as seen in the case of 
 `J=1`, `Mi=1` , `Mj=-1` , and `beta=0`. This results in division by `sin(0)`, which equals 0.